### PR TITLE
Update external.yml

### PR DESCRIPTION
**Pull request by Jubichon**
Add new workshop : unleash the power of Azure Open AI

### CODE REVIEW

File: packages/database/external.yml [hunk 1]
```diff
@@ -303,3 +303,19 @@
   audience: students, pro devs
   level: intermediate
   tags: github copilot, node.js, azure, express, javascript, typescript, app service, cosmos db, github actions, github codespaces
+
+  - title: Unleash the Power of Azure Open AI
```
Comment (sinedied) [inline packages/database/external.yml:307]: wrong indent

File: packages/database/external.yml [hunk 2]
```diff
@@ -303,3 +303,19 @@
...
+  url: https://microsoft.github.io/moaw/workshop/gh:microsoft/azurechat/main/docs/
```
Comment (sinedied) [inline packages/database/external.yml:308]: link does not work: it's not a workshop in MOAW format, so it should be a link to a website

File: packages/database/external.yml [hunk 3]
```diff
@@ -303,3 +303,19 @@
...
+  github_url: https://github.com/microsoft/azurechat
+  language: en
+  last_updated: 2023-10-03
+  type: workshop
+  description: Discover how to deploy a private chat tenant in their Azure Subscription, with a familiar user experience and the added capabilities of chatting over your data and files with Azure Chat Solution Accelerator powered by Azure Open AI Service.
+  authors:
+    - Thivy
+  contacts:
+    - '@thivy' https://github.com/thivy
```
Comment (sinedied) [inline packages/database/external.yml:317]:
```suggestion
    - '@thivy'
```
must be either one the other, can't be both

**Review by sinedied — CHANGES_REQUESTED** [review]

**Comment by sinedied** [comment]
build is broken, the indent doesn't seem to be fixed

**Comment by Jubichon** [comment]
@sinedied Correction done

**Pull request merged**